agrona/src/main/java/org/agrona/concurrent/ShutdownSignalBarrier.java

```diff
@@ -36,7 +36,11 @@
     {
         for (final String signalName : SIGNAL_NAMES)
         {
-            Signal.handle(new Signal(signalName), (signal) -> LATCHES.forEach(CountDownLatch::countDown));
+            final var previous = Signal.handle(new Signal(signalName), (signal) -> {});
```

Comment:
Project uses Java 8 as a minimum so `var` will not compile.

Reply:
fixed, sorry about that.